### ISSUE TITLE
when to update this library to apply glide v4 ?

### ISSUE THREAD

**Issue by yanxinmiao** [wasabeef/glide-transformations#78]
very expected...

**Comment by iielse** [comment]
![image](https://user-images.githubusercontent.com/7521943/29306946-4e7db154-81d2-11e7-8d30-6560ea9ad95b.png)

![image](https://user-images.githubusercontent.com/7521943/29306948-537f4d7a-81d2-11e7-9624-f29dfeb86a74.png)

**Comment by vpratfr** [comment]
Workaround: Subclass all required transformations

```
/**
 * To have Glide 4 compatibility
 * See: https://github.com/wasabeef/glide-transformations/issues/73
 */
class BlurTransformation(context: Context?, radius: Int) 
    : jp.wasabeef.glide.transformations.BlurTransformation(context, radius) {

    override fun transform(context: Context?,
                           resource: Resource<Bitmap>?, 
                           outWidth: Int, 
                           outHeight: Int): Resource<Bitmap> = transform(resource, outWidth, outHeight)

    override fun updateDiskCacheKey(md: MessageDigest) {
        md.update("blur".toByteArray())
    }
}
```

**Comment by wasabeef** [comment]
@yanxinmiao 
@iielse 
@vpratfr 

Just now, I'm preparing that. So I'll release the new version for 4.0.0 next week.

**Comment by yanxinmiao** [comment]
@wasabeef very good !!!

**Comment by wangshaolei** [comment]
@wasabeef nice~  @yanxinmiao @iielse @vpratfr now i implement with RoundedImageView...

**Comment by beartistic** [comment]
+1

**Comment by songjipo** [comment]
@wasabeef nice

**Comment by AndSync** [comment]
@wasabeef  "next week" you said 11days ago is gone ,
why did not see you commit anything

**Comment by se-bastiaan** [comment]
You should be happy that @wasabeef is willing to update the library, this developer has no obligations to any of us and still updates the library so that we can keep using it. There are probably other things that require attention and are more important. We should have respect for that and just wait :). In the meantime we can try to use the workaround above or we could create our own solutions for the time being. That might a bit harder than just using this library, but that shouldn't stop you. See it as a chance to learn something new!

**Comment by wasabeef** [comment]
@yanxinmiao @iielse @vpratfr @wangshaolei @beartistic @songjipo @AndSync @se-bastiaan 

You guys, I just released glide-transformations 3.0.0. So could you check that?

**Comment by yanxinmiao** [comment]
OK ， thanks
